### PR TITLE
Csv

### PR DESCRIPTION
**Pull request by PhilBogatyrev**
Csv download links

### CODE REVIEW

File: website/templates/athleteCoachView.html [hunk 1]
```diff
@@ -5,6 +5,13 @@
 
 <h1>{{ athlete.first_name }} {{ athlete.last_name }}</h1>
 </br>
+
```
Comment (tymkiwdylan) [inline website/templates/athleteCoachView.html:8]: Do we want the coach to be able to download the Athlete's data?

**Review by tymkiwdylan — COMMENTED** [review]
It looks good but I wonder if it is a good idea to let coaches download the Athlete's data.

File: website/templates/athleteCoachView.html (hunk 1 above)
Comment (PhilBogatyrev) [reply website/templates/athleteCoachView.html:8]: good point, deleted the csv links from coach template

**Review by tymkiwdylan — APPROVED** [review]
All good

**Pull request merged**